osprey_worker/src/osprey/worker/lib/storage/__init__.py

```diff
@@ -0,0 +1,30 @@
+from enum import StrEnum, auto
+
+
+class ExecutionResultStorageBackendType(StrEnum):
+    """Type of store used for execution results."""
+
+    BIGTABLE = auto()
+    """
+    Bigtable execution result store
+    """
+
+    GCS = auto()
+    """
+    Google Cloud Storage execution result store
+    """
+
+    MINIO = auto()
+    """
+    Minio execution result store
+    """
+
+    PLUGIN = auto()
+    """
+    Execution result store that is sdefined via register_execution_result_store
+    """
+
+    NONE = auto()
+    """
+    Disable execution results from being stored
```

Comment:
it may be helpful to tell users what the consequences of using this are
```suggestion
    Disable execution results from being stored. This may cause certain elements of Osprey to break, such as the events stream and individual event details in the UI
```